src/containers/Dashboard/DashboardApi.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import {config} from "../../app/config";
+
+export async function getNodes(auth: string | null) {
+    const url = `${config.apiURL}/user/node`;
+
+    try {
+        const response = await axios.get(url, {
+            headers: {
+                authorization: auth,
+            },
+        });
+        return response;
+    } catch (err) {
+        return err;
+    }
+}
+
+export async function getDiskDetails(auth: string | null, nodeIdValue: number) {
+    const url = `${config.apiURL}/user/node/diskinformation/${nodeIdValue}`;
+
+    try {
+        const response = await axios.get(url, {
+            headers: {
```

Comment:
You can write a helper function that will return headers config. Then you won't have repeated code regarding authorization token and if the authorization key needs to change at some point, we can change it on one place only.

Example:
```
const response = await axios.get(url, {
   ...getAuthHeaders(auth),
   params: {...}
}
```